test/rubygems/test_gem_resolver.rb

```diff
@@ -95,81 +95,6 @@ def test_self_compose_sets_single
     assert_same index_set, composed
   end
 
-  def test_handle_conflict
-    a1 = util_spec 'a', 1
-
-    r1 = Gem::Resolver::DependencyRequest.new dep('a', '= 1'), nil
-    r2 = Gem::Resolver::DependencyRequest.new dep('a', '= 2'), nil
-    r3 = Gem::Resolver::DependencyRequest.new dep('a', '= 3'), nil
-
-    existing = Gem::Resolver::ActivationRequest.new a1, r1, false
-
-    res = Gem::Resolver.new [a1]
-
-    res.handle_conflict r2, existing
-    res.handle_conflict r2, existing
-    res.handle_conflict r3, existing
-
-    assert_equal 2, res.conflicts.length
-  end
-
-  def test_requests
-    a1 = util_spec 'a', 1, 'b' => 2
-
-    r1 = Gem::Resolver::DependencyRequest.new dep('a', '= 1'), nil
-
-    act = Gem::Resolver::ActivationRequest.new a1, r1, false
-
-    res = Gem::Resolver.new [a1]
-
-    reqs = Gem::Resolver::RequirementList.new
-
-    res.requests a1, act, reqs
-
-    assert_equal ['b (= 2)'], reqs.to_a.map { |req| req.to_s }
-  end
-
-  def test_requests_development
-    a1 = util_spec 'a', 1, 'b' => 2
-
-    spec = Gem::Resolver::SpecSpecification.new nil, a1
-    def spec.fetch_development_dependencies
-      @called = true
-    end
-
-    r1 = Gem::Resolver::DependencyRequest.new dep('a', '= 1'), nil
-
-    act = Gem::Resolver::ActivationRequest.new spec, r1, false
-
-    res = Gem::Resolver.new [act]
-    res.development = true
-
-    reqs = Gem::Resolver::RequirementList.new
-
-    res.requests spec, act, reqs
-
-    assert_equal ['b (= 2)'], reqs.to_a.map { |req| req.to_s }
-
-    assert spec.instance_variable_defined? :@called
-  end
-
-  def test_requests_ignore_dependencies
-    a1 = util_spec 'a', 1, 'b' => 2
-
-    r1 = Gem::Resolver::DependencyRequest.new dep('a', '= 1'), nil
-
-    act = Gem::Resolver::ActivationRequest.new a1, r1, false
-
-    res = Gem::Resolver.new [a1]
-    res.ignore_dependencies = true
-
-    reqs = Gem::Resolver::RequirementList.new
-
-    res.requests a1, act, reqs
-
-    assert_empty reqs
-  end
```

Comment:
Presumably these were added for a reason?  Are they going to be added back, or, what was the reason for removing the tests?